api/graphql/schema.graphql

```diff
@@ -1684,17 +1684,17 @@ type Mutation {
   publishPoolAdvertisement(id: ID!): PoolAdvertisement
     @guard
     @can(ability: "publish", find: "id", model: "Pool")
-  changePoolExpiryDate(
+  changePoolClosingDate(
```

Comment:
Just wondering, do you know why this was implemented with its own Mutation class (`ChangePoolClosingDate.php`) instead of the usual `@update(model: "Pool")` directive?

Reply:
These are a bit funny because the closing date is the only field you can update on a published pool.  Everything else is frozen.  That was probably the rationale on this mutation - if you control the parameters on a custom mutation you can prevent anything else from being changed.